tests/integration/test_config.py

```diff
@@ -0,0 +1,14 @@
+"""Integration tests for config loader."""
+
+import pytest
+
+from ols import config
+
+
+def test_load_non_existent_config():
+    """Test how loading of non-existent config is handled."""
+    global client
```

Comment:
`global client` is this line needed?

Reply:
actually it is not needed, fixed @onmete